tests/unit/rule_strings/pipeline.txt

```diff
@@ -0,0 +1,58 @@
+import linker.utilities.validation_utils as validation_utils
+rule all:
+    input:
+        final_output=['{snake_dir}/result.parquet'],
+        validation='{snake_dir}/input_validations/final_validator'
+rule:
+    name: "results_validator"
+    input: ['{snake_dir}/result.parquet']
+    output: touch("{snake_dir}/input_validations/final_validator")
+    localrule: True         
+    message: "Validating results input"
+    run:
+        for f in input:
+            validation_utils.validate_dummy_file(f)
+rule:
+    name: "step_1_python_pandas_validator"
+    input: ['{test_dir}/input_data1/file1.csv', '{test_dir}/input_data2/file2.csv']
```

Comment:
Is `.../input_data1/file1.csv` hardcoding acceptable? Is that how the things are actually bound in the container?

Reply:
>  Is that how the things are actually bound in the container?

There just aren't any containers at this point, until the next ticket.